### PR TITLE
Text issues in Installation page

### PR DESCRIPTION
**Pull request by hamza-siddiq**
## Summary
Fixes: #192 

Spelling and grammar fixes were made to the Installation page
## Testing Plan
Not needed
## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and
what additional work is required, if any.

```
[ ] Yes
```

### CODE REVIEW

**Comment by lwisne** [comment]
needs issue

**Comment by hamza-siddiq** [comment]
@lwisne thanks. Didn't know we needed them for documentation as well. No worries, just added the issue.

File: docs/onboarding/installation.md [hunk 1]
```diff
@@ -29,17 +29,17 @@ ironfish
 
 <br />
 
-When new versions are announced, you can update through NPM:
```
Comment (hughy) [inline docs/onboarding/installation.md:32]: I think the grammar is more correct as it is -- here `you can update through NPM` means that you are updating your installed Iron Fish. Changing to `you can update them` seems to suggest that you are updating the new versions themselves.

File: docs/onboarding/installation.md [hunk 2]
```diff
@@ -29,17 +29,17 @@ ironfish
...
 npm update -g ironfish
 ```
 
-Iron Fish is now ready to use. Follow the [next step](new_node.md) of the tutorial or jump directly to the [the CLI commands list](cli.md).
+Iron Fish is now ready to use. Follow the [next step](new_node.md) of the tutorial or jump directly to the [CLI commands list](cli.md).
 
 ## Alternative Installation Methods
 
-You can also get started with Iron Fish by using Docker or running from source. If you are using an Intel-based macOS system, we support Homebrew as an alternative installation method.
+You can also get started with Iron Fish by using Docker or running from the source. If you are using an Intel-based macOS system, we support Homebrew as an alternative installation method.
```
Comment (hughy) [inline docs/onboarding/installation.md:42]: I don't think that this change is necessary -- `from source` or `from source code` is very common phrasing

Comment (lwisne) [reply docs/onboarding/installation.md:42]: agree. this is making up work IMO.  Closing.

**Pull request closed without merging**